flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -299,6 +300,7 @@
     @Deprecated
     public static final ConfigOption<String> WATCH_HEARTBEAT_PAUSE =
             ConfigOptions.key("akka.watch.heartbeat.pause")
+                    .stringType()
```

Comment:
@matriv Did you consider to use the `durationType`? If the string format is the same and we can avoid side effects, we should definitely try to introduce the proper type for all intervals.

Reply:
Wouldn't that be a breaking change for affected options?

Reply:
Yep, it would, I would rather create a separate issue to track this change, WDYT?

Reply:
In any case I'd say that this would be outside the scope of this PR.